xmodule/xml_block.py

```diff
@@ -139,6 +140,40 @@ class XmlMixin:
 
     metadata_to_export_to_policy = ('discussion_topics',)
 
+    def _extract_prereq_url_name(self, prereq_usage_key):
+        """
+        Extract the url_name from a prerequisite usage key.
+        """
+        if isinstance(prereq_usage_key, str):
+            prereq_usage_key = UsageKey.from_string(prereq_usage_key)
+            return prereq_usage_key.block_id
+        else:
+            return None
+
+    def _add_prerequisite_to_xml(self, xml_object):
+        """
+        Add prerequisite information to sequential XML for export.
+        """
+        if self.category != 'sequential':
+            return
```

Comment:
Why are we modifying the whole `XmlMixin` instead of overriding the `add_xml_to_node` method in the `SequenceBlock`?